src/include/common/utils.h

```diff
@@ -67,5 +68,12 @@ static bool isLittleEndian() {
     return *(uint8_t*)&testNumber == 1;
 }
 
+static inline std::string parseCypherPatten(const std::string& pattern) {
```

Comment:
I think it is better to have a base regexpOperation class where we put all common methods.
So you can put `parseCypherPatten` and `copyToKuzuString` inside this class.